test/api/vote.test.js

```diff
@@ -201,20 +310,27 @@ describe('Vote API', () => {
     votes.length.should.equal(0, 'no vote should be added');
   });
 
-  it('should be possible to retrieve a vote', async function () {
-    const vote = await this.activeAlternative.addVote(this.user);
+  it('should be possible to retrieve a vote with hash', async function () {
+    const vote = await this.activeElection.addVote(this.user, []);
     const { body: receivedVote } = await request(app)
       .get('/api/vote')
       .set('Vote-Hash', vote.hash)
       .expect(200)
       .expect('Content-Type', /json/);
-
     receivedVote.hash.should.equal(vote.hash);
-    receivedVote.alternative._id.should.equal(String(vote.alternative));
-    receivedVote.alternative.election.should.deep.equal({
-      _id: String(this.activeElection.id),
-      title: this.activeElection.title,
-    });
+  });
+
+  it('should be possible to retrieve a vote with correct election', async function () {
```

Comment:
Can you add a test for the received `priotities` as well?
I.e that the posted priorites match the retrieved.